### ISSUE TITLE
Classes are not correctly exported in index.d.ts

### ISSUE THREAD

**Issue by ZihanChen-MSFT** [microsoft/flow2dts#22]
In `index.d.ts`  there are many exported items in this pattern:
```TypeScript
import AccessibilityInfo$f2tTypeof from "./Libraries/Components/AccessibilityInfo/AccessibilityInfo";
declare type AccessibilityInfo = typeof AccessibilityInfo$f2tTypeof;
export { AccessibilityInfo$f2tTypeof as AccessibilityInfo };
declare const $f2tExportDefault: { AccessibilityInfo: AccessibilityInfo; };
export default $f2tExportDefault;
```

In `AccessibilityInfo` there is:
```TypeScript
declare var AccessibilityInfo: { /* members */ };
declare const $f2tExportDefault: typeof AccessibilityInfo;
```

`AccessibilityInfo` is supposed to be both a value and a type,
but I suspect `index.d.ts` export it as value twice.

Here are `AccessibilityInfo` in `@types/react-native`:
```TypeScript
export interface AccessibilityInfoStatic { ... }
export const AccessibilityInfo: AccessibilityInfoStatic;
export type AccessibilityInfo = AccessibilityInfoStatic;
```

**Comment by ZihanChen-MSFT** [comment]
Yes the imported `AccessibilityInfo` is an object.

**Comment by ZihanChen-MSFT** [comment]
Fixed by changes from 1daa7f91 to b098aa46
